pelita/scripts/pelita_main.py

```diff
@@ -221,8 +221,8 @@ def long_help(s):
                            help='Maximum number of rounds to play.')
 game_settings.add_argument('--seed', type=int, metavar='SEED', default=None,
                            help='Initialize the random number generator with SEED.')
-game_settings.add_argument('--allow-squatting', type=bool, default=False,
-                           help='do not set a food lifetime')
+game_settings.add_argument('--allow-camping', const=True, action='store_const',
```

Comment:
default should be `False`, right?

Reply:
I think now the default is `None` which is fine.